client/package.json

```diff
@@ -181,7 +181,7 @@
   },
   "scripts": {
     "build": "NODE_ENV=production webpack --config config/webpack.client.prod.js",
-    "start": "export SERVER_URL=${SERVER_URL:-'http://localhost:8080'} DEV_PORT=${DEV_PORT:-3000} ; NODE_ENV=development webpack serve --config config/webpack.client.dev.js",
+    "start": "export SERVER_URL=${SERVER_URL:-'http://localhost:8080'} DEV_PORT=${DEV_PORT:-3000} ; NODE_ENV=development webpack serve --config config/webpack.client.dev.js --mode development",
```

Comment:
`webpack.client.dev.js` already specifies `mode: "development"` so this seems redundant?

Reply:
It indeed did not effect the hot-reloading. I assume we have a one-liner.